syntax = "proto3";

package com.example;

message StartStopwatchParams {
}
